model/src/test/java/io/syndesis/model/connection/ActionPropertySuggestionsTest.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.model.connection;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.junit.Test;
+
+import static io.syndesis.model.connection.ActionPropertySuggestions.ActionPropertySuggestion.Builder.of;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class ActionPropertySuggestionsTest {
+
+    private final static String JSON = "{\"property1\":[{\"displayValue\":\"1displayValue1\",\"value\":\"1value1\"},{\"displayValue\":\"1displayValue2\",\"value\":\"1value2\"}],\"property2\":[{\"displayValue\":\"2displayValue1\",\"value\":\"2value1\"},{\"displayValue\":\"2displayValue2\",\"value\":\"2value2\"}]}";
```

Comment:
actually, I prefer to have this kind of test data in an external, pretty-printed resource file as its easier to update and understand.